FMB/FiniteModelBuilder.cpp

```diff
@@ -2588,8 +2588,6 @@ bool FiniteModelBuilder::SmtBasedDSAE::increaseModelSizes(DArray<unsigned>& newS
       }
       _smtSolver.add(sum == _context.int_val(_lastWeight));
 
```

Comment:
@selig - I believe this is your domain. Seems like you check Z3 twice and throw away the first result?

Reply:
Yeah, probably leftover from debugging ! I don't think it will be a performance hit as z3 will cache the result as nothing is added but deleting the line makes sense as it's not doing anything!